index.html

```diff
@@ -186,6 +186,20 @@ <h2>Scope</h2>
           respectively.
         </p>
 
+        <p>
+          Depending on community discussions and demands, this Working Group may also decide to submit the EPUB 3.3, 
+          the EPUB Reading Systems 3.3, and the EPUB Accessibility 1.1 specifications to 
+          <a href="https://www.iso.org/committee/45020.html">ISO/IEC JTC1</a> to be published as ISO/IEC standards.
+          This submission would follow the <a href="https://www.w3.org/2010/03/w3c-pas-submission.html">PAS Submission process</a>.
```

Comment:
suggest s/This submission/If the decision is to proceed with a submission, this/

Reply:
+1